Getting Started Cleanup (#2466)
truenas/documentation: content/SCALE/GettingStarted/Install/ConsoleSetupMenuScale.md

The Console setup menu (CSM) displays at the end of the <file>iso </file> installation process and after the system boots up.
You can access this menu to administer the TrueNAS system if it has a keyboard and monitor.

> By default, TrueNAS does not display the Console setup menu when you connect via SSH or the web shell.

Reviewer: ```suggestion
By default, TrueNAS does not display the Console setup menu with SSH or web shell connections.
```
